src/pages/user/authority/AuthorityCard.tsx

```diff
@@ -27,7 +40,7 @@ const AuthorityCard: React.FC<AuthorityCardProps> = ({ authority }) => {
     <Box>
       <StyledBoxContent>
         <div>{authorityName && authorityName.value}</div>
-        <div>[Siste publikasjon]</div>
+        <div>{publications[0]}</div>
```

Comment:
Ikke sikkert at dette er siste publikasjon. publications må isåfall sorteres på dato.

Reply:
tar med dato sammen med title, og sorterer (naivt) på dato